doc/whats_new/v1.5.rst

```diff
@@ -53,6 +53,13 @@ Changelog
   platform-agnosticity of the estimator.
   :pr:`29612` by :user:`Olivier Grisel <ogrisel>`.
 
+:mod:`sklearn.impute`
```

Comment:
Sorry, I was a bit optimistic that we merge before 1.5.2 that is going to be release this week.

Could you move this entry in 1.6.0 at the end.

Reply:
Done